### ISSUE TITLE
Question about perceptual loss

### ISSUE THREAD

**Issue by BeepBump** [xinntao/BasicSR#145]
Hi, when using perceptual loss, we need a pretrained vgg-19 network.
And I want to ask that is the pretrained model u offered trained to apply for image classification?
Then, why u choose vgg-19 image classification network to define perceptual loss?

**Comment by young666** [comment]
Vgg-19 is a common Feature Extractor Network, Perceptual loss does calculate the loss of feature, so how could we could get the features to get the losses? Yeah, Vgg-19 help us extract the features from image. You better read the paper more carefully, they have already used Vgg-19 from SRGAN. ESRGAN make a slightly different that the features have been extracted before the activation.
